components/raftstore/src/store/config.rs

```diff
@@ -87,6 +87,9 @@ pub struct Config {
     // When the approximate size of raft log entries exceed this value,
     // gc will be forced trigger.
     pub raft_log_gc_size_limit: Option<ReadableSize>,
+    /// the maximum raft log numbers that applied_index can be ahead of
+    /// persisted_index.
+    pub max_apply_unpersisted_log_limit: u64,
```

Comment:
How to set the value properly? Or basically what are the implications of a large limit config?
I'm thinking the major issue is memory consumption. So if we can use the quota of the extra memory consumption for the queued unresisted logs to limit it, it may be more precise and also easy to config.

Reply:
I expect this config is only used temporarily before this feature is GAed. We will auto-adjusted(or remove this config) after we can automatically handle the memory control.